components/error_catcher.rst

```diff
@@ -0,0 +1,205 @@
+.. index::
+   single: Error
+   single: Exception
+   single: Components; ErrorCatcher
+
+The ErrorCatcher Component
+==========================
+
+    The ErrorCatcher component converts PHP errors and exceptions into other
+    formats such as JSON and HTML and renders them.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/error-catcher
+
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+The ErrorCatcher component provides several handlers and renderers to convert
+PHP errors and exceptions into other formats easier to debug when working with
+HTTP applications.
+
+.. TODO: how are these handlers enabled in the app? (Previously: Debug::enable())
```

Comment:
Yes, they are enabled by `Debug::enable()`, but we're still debating the correct namespace in https://github.com/symfony/symfony/pull/32303